### DISCUSSION TITLE
Welcome to iptv-checker Discussions!

### DISCUSSION THREAD

**Discussion by zmisgod** [zhimin-dev/iptv-checker#59] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by zmisgod** [comment]
欢迎有什么建议或者意见都可以在这里畅所欲言

**Reply by gittanker** [reply]
创建定时任务返回400.无法创建定时任务![Screenshot_2026-07-16-14-26-56-40_14ec16e9ffebba16df46eb69fe56ba85.jpg](https://github.com/user-attachments/assets/4c5d6a56-d891-4d67-add8-c87e2f65670f)

**Reply by zmisgod** [reply]
已修复，请更新v4.7.1版本
